catalyst_voices/lib/pages/registration/account_completed/account_completed_panel.dart

```diff
@@ -0,0 +1,274 @@
+              children: [
+                const _TitleText(),
+                const SizedBox(height: 10),
+                Column(
+                  children: <Widget>[
+                    _SummaryItem(
+                      image: VoicesAssets.images.registrationSummaryKeychain,
+                      title: context.l10n.registrationCompletedKeychainTitle,
+                      info: context.l10n.registrationCompletedKeychainInfo,
+                    ),
+                    BlocSelector<RegistrationCubit, RegistrationState, String>(
+                      selector: (state) =>
+                          state.walletLinkStateData.selectedWallet?.wallet
+                              .name ??
+                          '',
+                      builder: (context, walletName) {
+                        return _SummaryItem(
+                          image: VoicesAssets.images.registrationSummaryWallet,
+                          title: context.l10n
+                              .registrationCompletedWalletTitle(walletName),
+                          info: context.l10n
+                              .registrationCompletedWalletInfo(walletName),
+                        );
+                      },
+                    ),
+                    BlocSelector<RegistrationCubit, RegistrationState,
+                        List<AccountRole>>(
+                      selector: (state) =>
+                          state.walletLinkStateData.selectedRoles?.toList() ??
+                          [],
+                      builder: (context, roles) {
+                        return _SummaryItem(
+                          image: VoicesAssets.images.registrationSummaryRoles,
+                          title: context.l10n.registrationCompletedRolesTitle,
+                          info: context.l10n.registrationCompletedRolesInfo,
+                          footer: _RolesFooter(roles),
+                        );
+                      },
+                    ),
+                  ].separatedBy(const SizedBox(height: 10)).toList(),
+                ),
+              ],
+            ),
+          ),
+        ),
+        const _NextStep(),
+        const SizedBox(height: 10),
+        _OpenDiscoveryButton(
+          onTap: () {
+            Navigator.pop(context);
+            const DiscoveryRoute().go(context);
+          },
+        ),
+        const SizedBox(height: 10),
+        _ReviewMyAccountButton(
+          onTap: () {
+            Navigator.pop(context);
+            const AccountRoute().go(context);
+          },
+        ),
+      ],
+    );
+  }
+}
+
+class _RolesFooter extends StatelessWidget {
+  final List<AccountRole> roles;
+
+  const _RolesFooter(this.roles);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        ...roles.map(
+          (role) => Row(
+            children: [
+              Text(
+                '1x',
```

Comment:
Shouldn't we need to localize "1x"?

Reply:
@dtscalac preferred it not localised https://github.com/input-output-hk/catalyst-voices/pull/996#discussion_r1798989646

Reply:
The only reason to localize it would be if we wanted it reusable but then it would need to support plurals. For now I would keep it simple.